src/python/pants/python/python_setup.py

```diff
@@ -20,6 +20,8 @@ class PythonSetup(Subsystem):
 
     options_scope = "python-setup"
 
+    _DEFAULT_MANYLINUX_UPPER_BOUND = "manylinux2014"
```

Comment:
Is it true that this should be removed once `--resolver-use-manylinux` is removed? If so, might help to add a `TODO` to remove this when that happens.

Reply:
Nope, it stays. It should probably be inlined when its two usages collapse to one, but I don't think I need a note on that.